README.template.md

```diff
@@ -0,0 +1,6 @@
+# Roadrunner Rails
+
+This build monitor should be up and running as soon as you set up travis!
+[![Build Status](https://travis-ci.org/department-of-veterans-affairs/roadrunner-rails.svg?branch=master)](https://travis-ci.org/department-of-veterans-affairs/roadrunner-rails)
+
+Tell people about your project!
```

Comment:
Yeah, didn't put too much effort into the template. 

Not sure exactly what you mean by cutting off the top instructions, but add what you think should go there. That readme template looks solid.

All instances of `roadrunner-rails` will be replaced with `your-project-name` in this URL, so the travis urls should work. You just have to setup the job.